src/components/Referrals/ReferralHeroSection/ReferralHeroSection.spec.tsx

```diff
@@ -54,12 +55,17 @@ describe('ReferralHeroSection', () => {
       expect(getByText(t('referral_hero_section.subtitle'))).toBeInTheDocument()
     })
 
-    it('should display the three steps', () => {
-      const { getByTestId, getByText } = renderReferralHeroSection()
+    it('should display the steps container with hidden steps initially', () => {
```

Comment:
This `it` talks about hidden steps, but there's no assertion to check that they're hidden, would you mind adding the assertions related to that condition?